docs/modules/ROOT/pages/adr/0043-strimzi-operator-for-kafka.adoc

```diff
@@ -0,0 +1,407 @@
+
+[cols="1,1,1"]
+|===
+|Requirements |https://strimzi.io/[Strimzi] |https://github.com/confluentinc/confluent-kubernetes-examples[Confluent Operator]
+
+|KRaft Mode Support |✅ |✅
+
+|Schema Registry Integration |❌ (Apicurio) |✅ (Confluent)
+
+|Topic and User Management |✅ (Native CRDs) |✅
+
+|Metrics Export |✅  |✅
+
+|Multi-AZ Support |✅ (Rack awareness) |✅ (Rack awareness)
+
+|2-AZ Support |❌ (reconfiguration needed) |✅ (automatic observer promotion)
+
+|Automated Rebalancing |✅ (Cruise Control) |✅ (Self-balancing clusters)
+
+|TLS/mTLS Support |✅ |✅
+
+|Open Source |✅ (Apache 2.0) |❌ (Proprietary)
+
+|Active Development |✅ |✅
+
+|Grafana Dashboards |✅ (Community maintained) |✅ (Community maintained)
+
+|Commercial Support Available |✅ (Red Hat, SPOUD) |✅ (Confluent)
+
+|Maturity |✅ CNCF Sandbox, Production-ready since 2018 |✅ Widely used in enterprise
+
+|===
+
+Additional Notes:
+
+* **Strimzi** is a CNCF sandbox project with strong community backing and Red Hat support. It has been production-ready since 2018 and is used by numerous enterprises worldwide. It provides the most comprehensive mature Kubernetes-native approach with extensive CRDs for all Kafka resources available as open-source.
+* **Confluent Operator** requires a license for production use and ties users to Confluent's ecosystem, which conflicts with our preference for open-source flexibility. If customers specifically require Confluent features, we can consider it as a separate offering.
+
+**Not Considered:**
+
+* **https://github.com/adobe/koperator[Koperator]** (Banzai Cloud): Limited recent development activity and lacks some operational features like Cruise Control integration. Not actively maintained.
+* **https://github.com/bitnami/charts/tree/main/bitnami/kafka[Bitnami Helm Chart]**: Lacks advanced operational features like topic/user management CRDs and automated rebalancing. More suitable for development environments than production.
+
+
+== Decision
+
+We will use https://strimzi.io/[Strimzi Kafka Operator] to provide Apache Kafka as a managed service.
+
+The platform will consist of the following components:
+
+=== Core Components
+
+**Kafka Cluster (Strimzi Operator)**::
+
+* Manages Kafka brokers and KRaft controllers
+* Provides Kubernetes-native resources (CRDs) for Kafka, KafkaTopic, KafkaUser
+* Integrates Cruise Control for automated cluster rebalancing
+* Supports rolling updates with zero downtime
+* Version support follows https://github.com/strimzi/strimzi-kafka-operator/blob/main/KAFKA_VERSION_SUPPORT.md[Strimzi's version support policy]
+
+**Schema Registry (Apicurio Registry)**::
+
+* Manages Avro, Protobuf, and JSON schemas
+* Compatible with Confluent Schema Registry API
+* Integrated with Kafka for schema storage backend
+
+**Web UI (AKHQ)**::
+
+* Provides operational visibility into Kafka clusters
+* Allows browsing topics, consumer groups, and messages
+* Facilitates debugging and troubleshooting
+* Read-only mode for production environments
```

Comment:
Might make sense to add a small note to the "Implementation Notes" section for RO mode.